Vineyard/Vineyard/TodoView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  TodoView.swift
+//  Vineyard
+//
+//  Created by Sacchit Mittal on 10/3/24.
+//
+
+import SwiftUI
+
+struct TodoView: View {
+    @State private var viewModel = GroupsListViewModel()
+    
+    var body: some View {
+        NavigationStack {
+            ScrollView {
+                VStack(spacing: 10){
+                    ForEach($viewModel.user.groups) { $group in
```

Comment:
We should access the resolutions through user.allProgress instead. This is because the group holds template resolutions, and then the user specifies the specific values for the frequency and quantity that they want.

Reply:
Yeah makes sense, it didn't feel right to me when I put that down